client/lib/cart-values/index.js

```diff
@@ -65,6 +65,10 @@ function isFree( cart ) {
 	return cart.total_cost === 0 && ! cartItems.hasFreeTrial( cart );
 }
 
+function isFreeTrial( cart ) {
```

Comment:
What do you think about naming this something like `hasSingleFreeTrial`? Otherwise it sounds like a function that should be used with a single cart item.